plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -153,4 +157,19 @@ public IcebergNessieCatalogConfig setCompressionEnabled(boolean compressionEnabl
         this.compressionEnabled = compressionEnabled;
         return this;
     }
+
+    public Map<String, String> getConfigMap()
```

Comment:
Pls move this method to `IcebergNessieCatalogModule`
The _config_ class doesn't need to be dependent on external libraries.

Reply:
good suggestion. Moved.